pages/content/amp-dev/documentation/courses/beginning-course/course-introduction@pt_BR.md

```diff
@@ -0,0 +1,108 @@
+
+Abra <a href="https://glitch.com/edit/#!/nosy-leech" target="_blank">esse projeto</a> para começar. Clique no botão “Remix to edit” no lado superior direito para criar um novo projeto que você pode editar. Você pode continuar a usar este editor para este e futuros cursos. Cada curso futuro também lhe dará a oportunidade de começar com uma versão de referência da solução até aquele ponto.
+
+Você não precisa usar o Glitch para concluir esses treinamentos. No entanto, parte do código necessário para completar os exercícios está contido apenas nos exemplos de Glitch. Se desejar usar outro editor, ainda pode ser necessário usar os exemplos de Glitch para encontrar este código.
+
+## Configurando o AMP Validator <a name="setting-up-the-amp-validator"></a>
+
+Para detectar erros em nossas páginas AMP, temos uma ferramenta valiosa ao nosso alcance: o validador de AMP. Escrever páginas AMP válidas é a chave para acessar todos os benefícios do framework. O validador de AMP pode ser acessado de duas maneiras: por meio de uma extensão do Chrome ou adicionando um parâmetro ao nosso URL para que nossa página AMP use o validador integrado. Para os fins deste curso, sugerimos que você use a extensão do Chrome, pois é mais fácil de usar e acessar enquanto você constrói seu site.
+
+- Para instalar a extensão do Chrome, visite o link [aqui](https://chrome.google.com/webstore/detail/amp-validator/nmoffdblmcmgeicmolmhobpoocbbmknc/related?hl=pt_br).
+
+- Para usar o validador de AMP integrado, adicione `#development=1` no final do URL da página AMP e abra o console do desenvolvedor em seu navegador para ver os resultados. Você não precisa adicionar esse parâmetro se estiver usando a extensão do Chrome.
+
+## O que vamos construir
+
+Ao longo de nossos três cursos, você construirá um site para a Loja de Bicicletas de Queijo do Chico. O Chico desenvolveu uma bicicleta revolucionária feita inteiramente de queijo. A demanda por bicicletas novas é tão alta que o Chico precisa criar um site o mais rápido possível. Quando terminarmos esses cursos, o site do Chico se parecerá com este:
+
+{{ image('/static/img/courses/beginner/image14.png', 311, 550, align='center third', caption='Como irá ficar o site no final do Curso Avançado') }}
+
+Você pode clicar nesse<a href="https://nice-consonant.glitch.me/" target="_blank">link</a> para ver uma prévia ao vivo. Dê uma olhada no site. Temos vídeos, formulário de inscrição, carrossel de imagens e maneiras de compartilhar nosso site nas redes sociais. Abra o menu de navegação clicando no ícone formado por três linhas (também chamado de “ícone do menu de hambúrguer”) no canto superior esquerdo. Assim que o menu se expandir, clique no link “Nossos Produtos” para navegar até uma lista de produtos. Tente classificar a lista de produtos por preço e filtrar a lista de produtos por categoria de produto.
```

Comment:
need a space before the link.

I also noticed this is "neste link" in the Expert mode. I don't know Portuguese, so I don't know which of these is more idiomatic!

Reply:
Excellent!
Done!